.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots/Terminal output**
+If applicable, add screenshots to help explain your problem.
+
+**Environment Details (please complete the following information):**
+ - Host OS: [e.g. output from uname -a]
```

Comment:
Should we leave the square brackets? Also I had a hard time preventing a #<span/>1 in the uname ouput from linking to the #1 issue. Had to write `#<span/>1`. Maybe there are easier ways, but I do not want it to appear as code.